src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -413,6 +449,83 @@ void CodeGen::genHWIntrinsic_R_R_RM_I(GenTreeHWIntrinsic* node, instruction ins)
     }
 }
 
+void CodeGen::genJumpTableFallback(
+    GenTreeHWIntrinsic* node, regNumber targetReg, regNumber op1Reg, regNumber op2Reg, regNumber op3Reg)
+{
+    assert(op1Reg != REG_NA);
+    assert(op2Reg != REG_NA);
+    NamedIntrinsic  intrinsicID    = node->gtHWIntrinsicId;
+    var_types       baseType       = node->gtSIMDBaseType;
+    emitAttr        attr           = (emitAttr)node->gtSIMDSize;
```

Comment:
I don't think it makes a significant difference but in many places the `EA_ATTR` macro is used for this.